test/everypolitician/popolo/person_test.rb

```diff
@@ -155,6 +155,13 @@ def test_person_equality_based_on_id
     person2 = Everypolitician::Popolo::Person.new(id: '123', name: 'Bob', gender: 'male')
     organization = Everypolitician::Popolo::Organization.new(id: '123')
     assert_equal person1, person2
+  end
+
+  def test_person_equality_based_on_class
```

Comment:
This test is still testing two quite different things: 

1. Testing that two people with the same ID but different names are equal
2. Testing that a Person and an Organisation with the same IDs are not equal